src/job_description_feature_extraction.py

```diff
@@ -105,14 +161,12 @@ def _build_idf_dict(tfidf_vectorizer):
     return idf_dict
 
 
-def _vectorize(job_description):
+def _vectorize(job_description_list, tfidf_vectorizer=TfidfVectorizer()):
```

Comment:
rename this, `_vectorize` doesn't communicate any information about TF/IDF (which is its main function)